forms-flow-web-root-config/src/index.ejs

```diff
@@ -6,8 +6,8 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>formsflow.ai</title>
   <script src="/config/config.js"></script>
-  <link rel="stylesheet" href="https://unpkg.com/bpmn-js-properties-panel@1.3.0/dist/assets/element-templates.css">
```

Comment:
why have we deleted a css import ? is it showing issues ?

Reply:
Now new style are added this is specific to that version